app/code/Magento/CatalogImportExport/Model/Export/Product.php

```diff
@@ -1024,12 +1026,10 @@ protected function collectRawData()
                     unset($data[$itemId][$storeId][self::COL_ADDITIONAL_ATTRIBUTES]);
                 }
 
-                if (!empty($data[$itemId][$storeId]) || $this->hasMultiselectData($item, $storeId)) {
-                    $attrSetId = $item->getAttributeSetId();
-                    $data[$itemId][$storeId][self::COL_STORE] = $storeCode;
-                    $data[$itemId][$storeId][self::COL_ATTR_SET] = $this->_attrSetIdToName[$attrSetId];
-                    $data[$itemId][$storeId][self::COL_TYPE] = $item->getTypeId();
-                }
+                $attrSetId = $item->getAttributeSetId();
```

Comment:
Does this also change the behavior for non-image multi-select data also? Do we have a test covering this?

Reply:
It covers by Magento/CatalogImportExport/Model/Export/ProductTest::testExport() for instance. Exported product has multiselect data.